RecoLocalCalo/HGCalRecProducers/python/HGCalRecHit_cfi.py

```diff
@@ -75,6 +75,13 @@
     HGCHEF_fCPerMIP = HGCalUncalibRecHit.HGCHEFConfig.fCPerMIP,
     HGCHEB_keV2DIGI = hgchebackDigitizer.digiCfg.keV2MIP,
     HGCHEB_isSiFE   = HGCalUncalibRecHit.HGCHEBConfig.isSiFE,
+    # don't produce rechit if detid is a ghost one
+    # smallest and largest ghost detid on positive side
+    GhostDetIdPosMin = 1162362881,
```

Comment:
run time errors are not surprisingly coming from here: there is no cms type assigned to the values